src/main/resources/swagger-ui/index.html

```diff
@@ -71,10 +71,10 @@
 <script src="./swagger-ui-standalone-preset.js"> </script>
 <script>
 window.onload = function() {
+  
   // Build a system
   const ui = SwaggerUIBundle({
-    spec: ${swaggerSpecification},
-    validatorUrl: null,
+    url: "http://petstore.swagger.io/v2/swagger.json",
```

Comment:
Thanks for the PR @radarsh! We should indeed keep the `validatorUrl: null`. Validation can not work because we are running swagger-ui locally.